.storybook/withMas.tsx

```diff
@@ -32,7 +32,6 @@ export const withMas = (
                 datascienceoverview: () =>
                   import("../locales/en/datascienceoverview.json"),
                 kafkaoverview: () => import("../locales/en/kafkaoverview.json"),
-                topic: () => import("../locales/en/topic.json"),
```

Comment:
don't delete topic.json file, I am adding all the topic related code in topic.json file

Reply:
Reverted